{"name": "使用无效凭证登录", "status": "passed", "steps": [{"name": "打开登录页面", "status": "passed", "start": 1743469579898, "stop": 1743469580372}, {"name": "输入错误的用户名和密码", "status": "passed", "start": 1743469580372, "stop": 1743469580552}, {"name": "验证错误消息", "status": "passed", "start": 1743469580552, "stop": 1743469580574}], "start": 1743469579898, "stop": 1743469580574, "uuid": "408adacf-f7c1-4ccd-af79-d0517367c748", "historyId": "469e1c7a7a786adeb6bc7642147fd103", "testCaseId": "469e1c7a7a786adeb6bc7642147fd103", "fullName": "tests.test_web_ui.TestLogin#test_failed_login", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "失败登录"}, {"name": "feature", "value": "登录功能"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_web_ui"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "P14s"}, {"name": "thread", "value": "10868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_web_ui"}]}